File: C#-Advanced/C#-OOP/02.InheritanceExercise/Restaurant/Beverage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Restaurant
{
    public class Beverage : Product
    {
        public Beverage(string name, decimal price, double mililiters) : base(name, price)
        {
            Mililiters = mililiters;
        }

        public double Mililiters { get; set; }
    }
}

File: Python-OOP/04Encapsulation-Exercise/04Restaurant/project/beverage/beverage.py
from project.product import Product


class Beverage(Product):
    def __init__(self, name, price, milliliters):
        super().__init__(name, price)
        self.milliliters = milliliters

    @property
    def milliliters(self):
        return self.__milliliters

    @milliliters.setter
    def milliliters(self, value):
        self.__milliliters = value

